.evergreen/config.yml

```diff
@@ -730,6 +734,19 @@ axes:
            SSL: "nossl"
            AUTH: "noauth"
 
+  - id: csfle
```

Comment:
Since we are only using `local` KMS I think it might be fine if we always just install `pymongocrypt` and run these tests - would you agree @ShaneHarvey ?
If this is the case, we don't need to add this new axis.